src/sql/workbench/api/common/sqlExtHostTypes.ts

```diff
@@ -324,6 +324,11 @@ export enum AzureResource {
 	Sql = 1
 }
 
+export class TreeItem extends vsExtTypes.TreeItem {
+	payload: IConnectionProfile;
+	providerHandle: string;
```

Comment:
It might be good to use a less generic name than `TreeItem` since it sounds like this is just for the OE tree